src/modelize/modelize_class.nit

```diff
@@ -312,6 +313,11 @@ redef class ModelBuilder
 
 		# Create all classes
 		for nclassdef in nmodule.n_classdefs do
+			if not nclassdef isa AStdClassdef then continue
+			self.build_a_mclass(nmodule, nclassdef)
+		end
+		for nclassdef in nmodule.n_classdefs do
+			if nclassdef isa AStdClassdef then continue
```

Comment:
the commit doc says it
https://github.com/privat/nit/commit/9bb7117c0e879f5ed164f7c06e027983aa6fe378

On Wed, Apr 8, 2015 at 9:44 PM, Alexandre Terrasa <notifications@github.com>
wrote:

> In src/modelize/modelize_class.nit
> <https://github.com/privat/nit/pull/1249#discussion_r27976034>:
>
> > @@ -312,6 +313,11 @@ redef class ModelBuilder
> >
> >  		# Create all classes
> >  		for nclassdef in nmodule.n_classdefs do
> > +			if not nclassdef isa AStdClassdef then continue
> > +			self.build_a_mclass(nmodule, nclassdef)
> > +		end
> > +		for nclassdef in nmodule.n_classdefs do
> > +			if nclassdef isa AStdClassdef then continue
>
> So you build_a_mclass in both case? Why do you need two loops?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/privat/nit/pull/1249/files#r27976034>.
>